scripts/helpers.ts

```diff
@@ -89,7 +89,6 @@ export const reservedKeywords = new Set([
   'throw',
   'while',
   'yield',
-  'delete',
```

Comment:
Was removing this intentional?

Reply:
ugh...No, but Design does have an icon named `delete`. We will need them to address the conflict. Great catch 🦅 👁️  :joy: